### ISSUE TITLE
[FE] Update Review Page (eCheck-in & Stand-alone Travel)

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#78733]
## Description
Based on input from the design team, there have been some changes to the Review Page

## Wireframes
[Wireframe for stand-alone travel](https://www.figma.com/file/pnR05o7NPJDS0KFUSQ0eE3/LoROTA-Travel-Reimbursement-%7C-Check-in?type=design&node-id=2453%3A3758&mode=design&t=Q3dCZINAZqiKA4b9-1)

[Wireframe for eCheck-in](https://www.figma.com/file/XYoHvDxgB2BpZIo2rrhFba/Check-in-%7C-Check-in?type=design&node-id=3237%3A809&mode=design&t=KzMQE20uyuvKuRXr-1) 

## Tasks
- Make changes to the review page per the wireframes **in the stand-alone Travel app**
- Make changes to the review page per the wireframes **in eCheck-in**

**Comment by loripusey** [comment]
@lee-delarm6 did @benbrasso-agile6 or anyone else validate this? Found a couple things:

1. The eyebrow is missing from the agreement page
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/86678742/9fcd1707-32f3-4a97-8a14-c9e40d5eca43)

2. Content under "What you're claiming" says "Mileage reimbursement only" but the wireframe says "Mileage-only reimbursement"
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/86678742/6ab23a03-ec5e-4abb-8486-dcea1ab617d5)

**Comment by benbrasso-agile6** [comment]
I have not validated this one, but I can check it

**Comment by lee-delarm6** [comment]
Sorry about that, on it!

**Comment by lee-delarm6** [comment]
The second one "mileage-only reimbursement" is dynamic data returned, still looking at the first one to look for the eyebrow in the wireframe

**Comment by lee-delarm6** [comment]
Ah, "Check In" is missing on the day-of checkin portion

**Comment by benbrasso-agile6** [comment]
eCheck-in wireframe w/ eyebrow - https://www.figma.com/file/XYoHvDxgB2BpZIo2rrhFba/Check-in-%7C-Check-in?type=design&node-id=3237%3A809&mode=design&t=dB9hmtLhBvPBv08D-1

**Comment by benbrasso-agile6** [comment]
In local, I selected "Details" link from ../complete/ and the incorrect check-in status appeared on the detail page. It had "Check in now" button. Is that just a reflection of local?

**Comment by lee-delarm6** [comment]
What do you mean you're seeing the wrong status? I think it should be showing that page (I didn't alter that link)

**Comment by benbrasso-agile6** [comment]
I'll catch up with you shortly

**Comment by loripusey** [comment]
@benbrasso-agile6 if you doing it locally, then it uses mock data & has no idea that you just checked in

**Comment by benbrasso-agile6** [comment]
Ah, okay. @lee-delarm6 did you update everything else? Ready to re-validate or close?

**Comment by lee-delarm6** [comment]
Eyebrow update was the only update made, JUST pushed it though. As a note on this ticket, the "check in now" button showing after checking in should just be a mock data issue

**Comment by benbrasso-agile6** [comment]
Screenshots? Then we'll close

**Comment by lee-delarm6** [comment]
![Screenshot 2024-04-01 at 4 47 09 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135059743/44f34fa0-3034-4efa-9d20-329a8654e792)

**Comment by benbrasso-agile6** [comment]
LGTM

**Comment by lee-delarm6** [comment]
![Screenshot 2024-04-01 at 5 19 24 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135059743/b30bb6fe-b367-4d31-b77d-b737667a9c7a)
![Screenshot 2024-04-01 at 5 17 37 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135059743/b478bcfc-84d6-423e-a9f2-d72ffec35f4b)


After talking with Ben and looking over the rest of the app, we came to the agreement on these being the only changes. No eyebrow needed for the shared page, beneficiary travel agreement is a shared page and outside the normal flow of the app (Ben will update wireframe)

**Comment by benbrasso-agile6** [comment]
Updated in Figma. LGTM.

**Comment by benbrasso-agile6** [comment]
Don't forget: 

check-in

`Mileage-only reimbursement`

travel app

`Mileage-only reimbursement for 1 appointment at {ParentStation}`

**Comment by lee-delarm6** [comment]
Oh shoot, I misunderstood, I thought you said both should be mileage only not `mileage-only` 👎  me

**Comment by lee-delarm6** [comment]
Fixed, for what I believe, to be the final time

**Comment by ndouglas** [comment]
>Mileage-only reimbursement for 1 appointment at {ParentStation}

Am I reading correctly that the "for" _should_ be present?

**Comment by benbrasso-agile6** [comment]
There's been a lot of back and forthing on this ticket. The wireframe links at the top of this ticket are still all correct. Let's validate against what's in the wireframes. 

Tl;dr within the travel app (not eCheck-in app), yes. 

When I did functional testing on the travel app yesterday, I passed every test case. So, not completely sure how we got so mixed up on this ticket 🤷

**Comment by lee-delarm6** [comment]
<img width="676" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNm9xQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9057fbd866ff3dcb62287fb15b29c627f7ca26d2/Screenshot%202024-04-02%20at%2011.13.28%E2%80%AFAM.png" alt="Screenshot 2024-04-02 at 11.13.28 AM.png" />

final-final-final.png :D
